index.html

```diff
@@ -257,6 +257,13 @@
             (isValid ? document.querySelector('form') : document.querySelector('section')).classList.add('hidden');
         };
 
+        function textareaInput(inputData) {
+            // first # may be wrongly encoded to %23 by safari/iOS; convert it back. see issue #16
+            inputData = inputData.replace(/%23/, '#')
```

Comment:
This is converting in the whole string 🤔 I wonder if we really need this hack, Will iOS app not generate links instead of copying to clipboard this raw data?

Reply:
> This is converting in the whole string 🤔

you're right, i made the hack more robust and only apply it when there is no single `#` (a correctly encoded string has exactly one `#). this is what 

> I wonder if we really need this hack

yeah, might we we get rid of the "copy+paste dialog" sooish anyways, but that is not clear, it is also useful for debugging. so, as long as we offer the "copy+paste dialog", it should work as good as it can, esp. as the issue only pops up when you already shared the link, we should take care.

i think the hack is easy enough, the whole "textarea input" can just be removed when we no longer want it